schemas/configuration/v4/zone_bundle.json.in

```diff
@@ -10,14 +10,20 @@
             "items": {
                 "type": "object",
                 "properties": {
+                    "coordinating_resources": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "resource.json"
+                        }
+                    },
                     "servers": {
                         "type": "array",
                         "items": {
                             "$ref": "server.json"
                         }
                     }
                 },
-                "required": ["servers"]
+                "required": ["coordinating_resources", "servers"]
```

Comment:
Coordinating resources should exists outside of servers. Originally they were attached to the catalog server, however it makes more sense for them to be a property of the zone because coordinating resources do not have a host server.